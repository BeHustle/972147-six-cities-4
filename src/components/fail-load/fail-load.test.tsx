import * as React from 'react';
import * as renderer from 'react-test-renderer';
import FailLoad from './fail-load';

it(`Render loading screen`, () => {
  const tree = renderer
    .create(<FailLoad />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
