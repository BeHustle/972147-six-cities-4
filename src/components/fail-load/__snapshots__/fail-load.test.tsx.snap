// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render loading screen 1`] = `
<h1>
  Error while loading. Try refresh page or
  <a
    href="/"
    style={
      Object {
        "color": "tomato",
        "textDecoration": "underline",
      }
    }
  >
    go to main
  </a>
</h1>
`;
